functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/simple/DisabledSmartTestingTestSelection.java

```diff
@@ -0,0 +1,24 @@
+package org.arquillian.smart.testing.ftest.simple;
+
+import java.util.List;
+import org.arquillian.smart.testing.ftest.TestBedTemplate;
+import org.arquillian.smart.testing.ftest.testbed.testresults.TestResult;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DisabledSmartTestingTestSelection extends TestBedTemplate {
+
+    @Test
+    public void should_execute_all_tests_when_disable_smart_testing_is_disabled() throws Exception {
+        // when
+        final List<TestResult> actualTestResults = project
+            .buildOptions()
+                .withSystemProperties("disableSmartTesting", "true")
+                .configure()
+            .build();
```

Comment:
Same as `should_execute_all_tests_when_smart_testing_is_disabled_irrespective_of_strategy` - there is no smart-testing extension configured so we just run regular build.